src/main/java/com/linecorp/armeria/client/tracing/http/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Tracing decorator for http client.
+ */
+package com.linecorp.armeria.client.tracing.http;
```

Comment:
How about merging this package into armeria.client.tracing, given that these packages have fairly small number of classes?